config/apiserver/role.yaml

```diff
@@ -82,3 +82,19 @@ rules:
   - get
   - list
   - watch
+- apiGroups:
```

Comment:
`make generate` will kill this again. You need to add magic comments for the generator:

```
// +kubebuilder:rbac:groups="kubernetes.crossplane.io",resources= providerconfigs,verbs=get;list;watch
```

Reply:
Oh good point. Fixed